apps/passport-client/pages/index.tsx

```diff
@@ -114,7 +124,7 @@ class App extends React.Component<object, AppState> {
       console.log(e);
     }
 
-    setTimeout(this.jobPollUser, 1000 * 60 * 5);
+    setTimeout(this.jobPollUser, 1000 * 10);
```

Comment:
maybe we can change this to be once a minute rather than the once a ten seconds you changed it to here